### ISSUE TITLE
Evals masking problems

### ISSUE THREAD

**Issue by xsawyerx** [Perl-Evozon/PearlBee#50]
If you're using `eval` to capture errors, you have to at least report it. When posting a new post, the `eval` just swallows the error without reporting.

**Comment by drforr** [comment]
Yep, and Emil's pull request handles those issues, I'll add that shortly. We're kind of focusing on the blogs.perl.org fork at the moment though.
